docs/03_configuration/README.md

```diff
@@ -94,19 +94,20 @@ So set `buildpath` to `/tmp/$USER` or `/dev/shm/$USER`, for example.*
 
 *(default: `<prefix>/sources`)*
 
-For most supported software, EasyBuild can **automatically download the source files** required for the installation. Before trying to download a source file, it will first check if it is already available.
+For most supported software, EasyBuild can **automatically download the source files** required for the installation.
+Before trying to download a source file, EasyBuild will first check if it is already present in the source path.
 
 The locations considered by EasyBuild when checking for available source files as well as the location to
 stored downloaded source files can be controlled via the ``sourcepath`` configuration setting.
 
 The `sourcepath` value is a colon (`:`) separated list of directory paths.
 Each of these paths will be considered in turn when checking for available source files,
 until one of them provides the desired source file. Searching for source files is done
-based on filename, and a couple of subdirectories as considered.
-For example, for a software packagamed '`Example'`, EasyBuild will consider locations like `<sourcepath>/e/Example/`, `<sourcepath>/Example/`, etc.
+based on filename, and a couple of subdirectories are considered.
+For example, for a software packagamed '`Example'`, EasyBuild will consider locations like `<sourcepath>/e/Example/`, `<sourcepath>/Example/`, and so on.
 
-The first path listed in `sourcepath` is the location where EasyBuild will store downloaded source files.
-It will keep them organized by software name through subdirectories.
+The first path listed in `sourcepath` will become the root directory where EasyBuild stores downloaded source files,
```

Comment:
Hmm, "root directory" may be confusing here (w.r.t. "root permissions").

Also, the "will become" feels a bit strangely worded. What's wrong with "is"?

Reply:
No idea what I was thinking here. I think I was getting tired.